packages/core/src/components/Button/Button.scss

```diff
@@ -161,6 +200,14 @@ Inverse buttons
   background-color: $color-background-inverse;
   border-color: $border-color-inverse;
 
+  &:disabled,
```

Comment:
If we wanted to follow BEM exactly, I _think_ we should have classes like this:
```
ds-c-button ds-c-button--primary ds-c-button--primary-disabled
ds-c-button ds-c-button--primary ds-c-button--primary-inverse
ds-c-button ds-c-button--primary ds-c-button--primary-active
```
but since that's a bit verbose and can get confusing with the HTML attributes and multiple button states (think inverse + disabled), I think it makes more sense to do this:
```
ds-c-button ds-c-button--primary ds-c-button--disabled
ds-c-button ds-c-button--primary ds-c-button--inverse
ds-c-button ds-c-button--primary ds-c-button--active
```